ovh/resource_ovh_iploadbalancing_http_frontend.go

```diff
@@ -176,6 +183,7 @@ func readIpLoadbalancingHttpFrontend(r *IpLoadbalancingHttpFrontend, d *schema.R
 	d.Set("display_name", r.DisplayName)
 	d.Set("port", r.Port)
 	d.Set("zone", r.Zone)
+	d.Set("redirect_location", r.RedirectLocation)
```

Comment:
as *string: should be 
if r.RedirectLocation != nil {
d.Set("redirect_location", *r.RedirectLocation)
}